### PR TITLE
Debian & FQCN support setup_repo, setup_replication, install_dbserver, init_dbserver and manage_dbserver

### PR DESCRIPTION
**Pull request by vibhorkum**
Modified the roles setup_repo, setup_replication, install_dbserver,init_dbserver, and manage_dbserver to support Debian. Also added FQCN wherever needed to maintain the backward compatibility with ansible 2.9

### CODE REVIEW

File: roles/setup_replication/tasks/upstream_node.yml [hunk 1]
```diff
@@ -7,7 +7,7 @@
   when:
     - hostvars[inventory_hostname].upstream_node_private_ip is defined
     - node.private_ip == hostvars[inventory_hostname].upstream_node_private_ip
-  loop: "{{ lookup('edb_devops.edb_postgres.pg_sr_cluster_nodes', wantlist=True) }}"
```
Comment (jt-edb) [inline roles/setup_replication/tasks/upstream_node.yml:10]: I would suggest to keep using the lookup function here. If we want to include only this tasks file, `pg_cluster_nodes` wont be defined.

File: roles/setup_repo/tasks/PG_RedHat_setuprepos.yml [hunk 2]
```diff
@@ -1,82 +1,65 @@
...
+    name: "{{ pg_rpm_repo_7 }}"
     state: present
-    key: "{{ pg_gpg_key_8 }}"
+  become: yes
   when:
-    - ansible_distribution_major_version == "8"
+    - ansible_distribution_major_version == "7"
     - pg_type == 'PG'
+
+- name: Install EPEL repo for RedHat 7
```
Comment (jt-edb) [inline roles/setup_repo/tasks/PG_RedHat_setuprepos.yml:43]: RedHat 8

File: roles/setup_repo/tasks/PG_RedHat_setuprepos.yml [hunk 3]
```diff
@@ -1,82 +1,65 @@
...
+  package:
+    name: "{{ pg_rpm_repo_8 }}"
     state: present
+  become: yes
   when:
     - ansible_distribution_major_version == "8"
     - pg_type == 'PG'
-  become: yes
 
 - name: Install EPAS repo for RedHat 7
```
Comment (jt-edb) [inline roles/setup_repo/tasks/PG_RedHat_setuprepos.yml:59]: Install EPAS repo for RedHat

**Review by jt-edb — COMMENTED** [review]
Few minor comments. Thank you!

File: roles/setup_repo/tasks/PG_RedHat_setuprepos.yml (hunk 3 above)
Comment (vibhorkum) [reply roles/setup_repo/tasks/PG_RedHat_setuprepos.yml]: Fixed

File: roles/setup_repo/tasks/PG_RedHat_setuprepos.yml (hunk 2 above)
Comment (vibhorkum) [reply roles/setup_repo/tasks/PG_RedHat_setuprepos.yml]: Fixed

File: roles/setup_replication/tasks/upstream_node.yml (hunk 1 above)
Comment (vibhorkum) [reply roles/setup_replication/tasks/upstream_node.yml]: fixed

**Review by jt-edb — APPROVED** [review]
LGTM

**Comment by vibhorkumar123** [comment]
Thank you! Merging!

**Pull request merged**